src/private/WRLDMapView.mm

```diff
@@ -76,6 +78,7 @@ @implementation WRLDMapView
     NSNumber* m_startDirection;
 
     std::unordered_map<WRLDOverlayId, id<WRLDOverlay>, WRLDOverlayIdHash, WRLDOverlayIdEqual> m_overlays;
+    std::unordered_map<int, WRLDBuildingHighlight*> m_buildingHighlights;
```

Comment:
Maybe WRLDBuildingHighlight could conform to WRLDOverlay protocol to avoid separate container here?
Polyons, Markers, Positioners etc contained in m_overlays